templates/tanstack-shell/app/components/LoginForm.tsx

```
@@ -0,0 +1,161 @@
+        e.preventDefault()
+        e.stopPropagation()
+        form.handleSubmit()
+      }}
+      className="space-y-6"
+    >
+      <form.Field
+        name="email"
+        validators={{
+          onChange: zodValidator(loginSchema.shape.email),
+        }}
+        children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)